library/systemd/test/test_helper.rb

```diff
@@ -23,7 +22,7 @@ def stub_systemctl(unit)
   end
 
   def stub_execute(success: true)
-    allow(Yast::Systemctl).to receive(:execute).and_return(
+    allow(Yast2::Systemctl).to receive(:execute).and_return(
```

Comment:
Reading this I have the doubt if `Systemctl` should also be under `Systemd` namespace. @jreidinger @imobachgs ?